tests/core/io/test_pck_packer.h

```diff
@@ -80,6 +81,22 @@ TEST_CASE("[PCKPacker] Pack empty with invalid key") {
 	ERR_PRINT_ON;
 }
 
+bool check_file(PCKReader &p_pck_reader, const String &p_source_path, const String &p_pack_path) {
```

Comment:
```suggestion
static bool check_file(PCKReader &p_pck_reader, const String &p_source_path, const String &p_pack_path) {
```
Keep local methods local